homeassistant/components/ekeybionyx/strings.json

```
@@ -0,0 +1,61 @@
+{
+  "config": {
+    "step": {
+      "pick_implementation": {
+        "title": "[%key:common::config_flow::title::oauth2_pick_implementation%]"
+      },
+      "choose_system": {
+        "data": {
+          "system": "System"
+        },
+        "data_description": {
+          "system": "System the webhooks should be set up for"
+        },
+        "description": "Please select the Ekey Bionyx system in which you want to set up the webhooks"
+      },
+      "webhooks": {
+        "description": "Please name your webhooks. You can set up to 5 webhooks. Leaving a name empty will skip the setup of that webhook.",
```

I stumbled while reading "You can set up …" till I finally realized it was "…up to 5 …" here.
We should use "configure" here instead to avoid that:

```suggestion
        "description": "Please name your webhooks. You can configure up to 5 webhooks. Leaving a name empty will skip the setup of that webhook.",
```